dot15d4-frame/src/repr/ie/nested.rs

```diff
@@ -143,56 +148,273 @@ impl TschSynchronizationRepr {
 /// A high-level representation of a TSCH Slotframe and Link Nested Information
 /// Element.
 #[derive(Debug)]
-#[cfg_attr(feature = "fuzz", derive(arbitrary::Arbitrary))]
 pub struct TschSlotframeAndLinkRepr {
-    /// The number of slotframes.
-    pub number_of_slotframes: u8,
+    /// The slotframe descriptors.
+    pub slotframe_descriptors: Vec<SlotframeDescriptorRepr, 3>,
```

Comment:
Can you add a TODO comment to remind us that we want to make this configurable in the future?